meta/src/main/proto/gravitino_meta.proto

```diff
@@ -114,4 +114,14 @@ message Group {
   string name = 2;
   repeated string roles = 3;
   AuditInfo audit_info = 4;
-}
\ No newline at end of file
+}
+
+message Role {
+  uint64 id = 1;
+  string name = 2;
+  repeated string privileges = 3;
+  string privilege_entity_identifier = 4;
+  string privilege_entity_type = 5;
+  map<string, string> properties = 6;
+  AuditInfo audit_info = 7;
```

Comment:
We don't record the users and groups. If we store the identifier of the roles in the groups and users instead of names. Because kv lack the transaction interface.
When we drop a role, we should remove a role from the users and groups.
I will add an special encoder for KVEntityStore in later grant pull request.